datadog/resource_datadog_sensitive_data_scanner_group.go

```diff
@@ -0,0 +1,243 @@
+package datadog
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/DataDog/datadog-api-client-go/v2/api/datadogV2"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+	"github.com/terraform-providers/terraform-provider-datadog/datadog/internal/utils"
+)
+
+func resourceDatadogSensitiveDataScannerGroup() *schema.Resource {
+	return &schema.Resource{
+		Description:   "Provides a Sensitive Data Scanner group resource.",
+		ReadContext:   resourceDatadogSensitiveDataScannerGroupRead,
+		CreateContext: resourceDatadogSensitiveDataScannerGroupCreate,
+		UpdateContext: resourceDatadogSensitiveDataScannerGroupUpdate,
+		DeleteContext: resourceDatadogSensitiveDataScannerGroupDelete,
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Description: "Name of the Datadog scanning group.",
+				Type:        schema.TypeString,
+				Required:    true,
+			},
+			"description": {
+				Description: "Description of the Datadog scanning group.",
+				Type:        schema.TypeString,
+				Optional:    true,
+			},
+			"product_list": {
+				Description: "List of products the scanning group applies.",
+				Type:        schema.TypeList,
+				Required:    true,
+				MaxItems:    4,
+				Elem: &schema.Schema{
+					Type: schema.TypeString,
```

Comment:
This looks great overall thanks! Is this field sorted in the backend/does order matter?

Reply:
Thanks a lot for the review! 

No the order doesn't matter. @pbeaudequin do you confirm that the order doesn't matter ?